pyMixtComp/python/setup.py

```diff
@@ -20,11 +30,21 @@
     keywords=["clustering", "mixture model", "heterogeneous", "missing data"],
     classifiers=[
         "Programming Language :: C++",
-        "Programming Language :: Python",
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
         "Topic :: Scientific/Engineering :: Artificial Intelligence"
     ],
     packages=["pyMixtComp"],
     include_package_data=True,
     package_data={"pyMixtComp": ["pyMixtCompBridge.so"], "": ["data/*.csv"]},
-    install_requires=["numpy", "pandas", "scikit-learn", "seaborn", "matplotlib", "scipy"],
+    python_requires='>={}'.format('.'.join(str(n) for n in py_min_version)),
+    install_requires=[
+        "numpy==1.23.4",
```

Comment:
I think that fixing version like that is too strict.

Reply:
But ensure stability across packages. I always thought doing that in python is important to avoid any unforeseen incompatibilities.

Reply:
It is important in your python project once you have installed all the packages you need but fixing version with == in a python package can create a lot of incompatibilities with other packages.